src/diffusers/loaders/single_file_utils.py

```diff
@@ -1223,14 +1240,34 @@ def create_diffusers_unet_model_from_ldm(
 
 
 def create_diffusers_vae_model_from_ldm(
-    pipeline_class_name, original_config, checkpoint, image_size=None, scaling_factor=None, torch_dtype=None
+    pipeline_class_name,
+    original_config,
+    checkpoint,
+    image_size=None,
+    scaling_factor=None,
+    torch_dtype=None,
+    model_type=None,
 ):
     # import here to avoid circular imports
     from ..models import AutoencoderKL
 
-    image_size = set_image_size(pipeline_class_name, original_config, checkpoint, image_size=image_size)
+    image_size = set_image_size(
+        pipeline_class_name, original_config, checkpoint, image_size=image_size, model_type=model_type
+    )
 
-    vae_config = create_vae_diffusers_config(original_config, image_size=image_size, scaling_factor=scaling_factor)
+    if "edm_mean" in checkpoint and "edm_std" in checkpoint:
+        if checkpoint["edm_mean"] is not None and checkpoint["edm_std"] is not None:
+            edm_mean = checkpoint["edm_mean"].to(dtype=torch_dtype) if torch_dtype else checkpoint["edm_mean"]
+            edm_std = checkpoint["edm_std"].to(dtype=torch_dtype) if torch_dtype else checkpoint["edm_std"]
+            vae_config = create_vae_diffusers_config(
+                original_config,
+                image_size=image_size,
+                scaling_factor=scaling_factor,
+                latents_mean=edm_mean.flatten().tolist(),
+                latents_std=edm_std.flatten().tolist(),
+            )
+    else:
+        vae_config = create_vae_diffusers_config(original_config, image_size=image_size, scaling_factor=scaling_factor)
```

Comment:
Could we do something like this here? 

```suggestion
    if model_type == "Playground":
        edm_mean = checkpoint["edm_mean"].to(dtype=torch_dtype) if torch_dtype else checkpoint["edm_mean"]
        edm_std = checkpoint["edm_std"].to(dtype=torch_dtype) if torch_dtype else checkpoint["edm_std"]

        edm_mean = edm_mean.flatten().tolist()
        edm_std = edm_std.flatten().tolist()

    else:
        edm_mean = None
        edm_std = None

    vae_config = create_vae_diffusers_config(
            original_config, 
            image_size=image_size, 
            scaling_factor=scaling_factor,
            latents_mean=edm_mean,
            latents_std=edm_std
    )
```